packages/@aws-cdk/aws-events-targets/lib/sqs.ts

```diff
@@ -40,6 +40,9 @@ export interface SqsQueueProps {
 export class SqsQueue implements events.IRuleTarget {
 
   constructor(public readonly queue: sqs.IQueue, private readonly props: SqsQueueProps = {}) {
+    if (typeof props.messageGroupId !== 'undefined' && !queue.fifo) {
+      throw new Error('MessageGroupId cannot be specified for non-FIFO queues.');
```

Comment:
remove period at the end of the message and change `MessageGroupId` to `messageGroupId`